libhoney/__init__.py

```diff
@@ -41,7 +41,7 @@
 def init(writekey="", dataset="", sample_rate=1,
          api_host="https://api.honeycomb.io", max_concurrent_batches=10,
          max_batch_size=100, send_frequency=0.25,
-         block_on_send=False, block_on_response=False):
+         block_on_send=False, block_on_response=False, xmit=None):
```

Comment:
total nitpick: since this'll be part of the API (hence hard to change later), may want to use `transmission` instead of `xmit` for clarity

Reply:
👍 Had to go with `transmission_impl` because `transmission` conflicts with the module of the same name.